### DISCUSSION TITLE
[igxGrid] [v13] [changelog] Do you have plan to add `width='auto'` ability to v13.x?

### DISCUSSION THREAD

**Discussion by loclv** [IgniteUI/igniteui-angular#12046] [Q&A] [answered]
I read your [changelog](https://github.com/IgniteUI/igniteui-angular/blob/master/CHANGELOG.md),
and find new ability to auto-size columns to the size of their cells and header content on initialization by setting width `auto` on `igxGrid`.

```html
<column width='auto' ...>
```

Do you have plan to add `width='auto'` ability to v13.x?

**Comment by zdrawku** [comment, marked as answer]
There is no plan to add this functionality as part of v13.x. Are you considering upgrading to 14.1.x?

**Comment by loclv** [comment]
> Are you considering upgrading to 14.1.x?

Perhaps in the distant future. :smile:

**Reply by zdrawku** [reply]
I understand. Please do consider that we strive to follow Angular supported version concept and release specifics, for example:
- Each new major release contains significant new features, as for the minor releases, they may contain smaller features, although the auto-sizing of the columns was not a small feature so it was added in a major release only.
- All Major releases are supported up to 18 months and due to the release schedule this means up to 3 major versions are supported. For more please refer to the [Angular releases page
](https://angular.io/guide/releases)![image](https://user-images.githubusercontent.com/11193764/197730427-74d12976-1483-43e8-bd9b-77b34929d253.png)

Sometimes we may extend the support of our versions or provide hot fixes based on the essence of the problem/requirement and the priority of it.
